_posts/2017-08-25-aug-25.md

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
I found 1 misspelled word in this sentence: 
- ---
	- *Suggested Replacements:* -o-, -i-